### DISCUSSION TITLE
Missing NXP i.MX RT pinctrl.dtsi files?

### DISCUSSION THREAD

**Discussion by nategreco** [zephyrproject-rtos/zephyr#53332] [Q&A] [answered]
Maybe this should be obvious, but I'm trying to find the file included here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/teensy4/teensy4-pinctrl.dtsi#L9

`mimxrt1062dvl6a-pinctrl.dtsi` isn't currently in the zephyrproject-rtos/zephyr repository - Should I be looking elsewhere?

**Comment by DerekSnell** [comment, marked as answer]
Hi @nategreco ,
These pinctrl files are in NXP's Hardware Abstraction Layer (HAL) repo, see https://github.com/zephyrproject-rtos/hal_nxp/blob/master/dts/nxp/nxp_imx/rt/mimxrt1062dvl6a-pinctrl.dtsi 

Best regards

**Reply by nategreco** [reply]
That's what I was missing, thank you.
